jetty-runner/src/main/java/org/eclipse/jetty/runner/Runner.java

```diff
@@ -123,7 +124,7 @@ public void addJars (Resource lib) throws IOException
                         if (lowerCasePath.endsWith(".jar") ||
                             lowerCasePath.endsWith(".zip"))
                         {
-                            URL url = item.getURL();
+                            URL url = item.getURI().toURL();
```

Comment:
Make `_classpath` a `List<URI>`, keep URIs all the way and only convert them to URLs at the very last moment and only if needed.

Reply:
done.